c/validation/adbc_validation.cc

```diff
@@ -1248,11 +1258,10 @@ void StatementTest::TestSqlIngestSample() {
               IsOkStatus(&error));
 
   ASSERT_THAT(AdbcStatementNew(&connection, &statement, &error), IsOkStatus(&error));
-  ASSERT_THAT(
-      AdbcStatementSetSqlQuery(
-          &statement, "SELECT * FROM bulk_ingest ORDER BY \"int64s\" ASC NULLS FIRST",
-          &error),
-      IsOkStatus(&error));
+  ASSERT_THAT(AdbcStatementSetSqlQuery(
+                  &statement, "SELECT * FROM bulk_ingest ORDER BY int64s ASC NULLS FIRST",
```

Comment:
we probably need a table-name-quote quirk or else just make these queries provided as part of the quirks, I expect

Reply:
well in this case it's not the table name, it's the column name that's the issue.

Reply:
Er, yes. But the point stands; we should start making the suite a little more flexible.

Reply:
true and fair. I'd rather keep that for a separate PR if that works for you?

Reply:
Yup - sounds good (do you mind filing an issue though?)

Reply:
Filed https://github.com/apache/arrow-adbc/issues/678